Content.Server/Nyanotrasen/Shuttles/Components/FTLWhitelistComponent.cs

```diff
@@ -0,0 +1,10 @@
+
+namespace Content.Server.Shuttles.Components;
+
+//Allows a grid to ignore FTL mass limit. Implemented to allow functioning of Spaceships, large shuttle grids/maps without issues.
+
+[RegisterComponent]
+public sealed class FTLWhitelistComponent : Component
+{
+    
+}
```

Comment:
```suggestion
}

```
Make sure your editor adds a newline at the end of a file. It doesn't matter here, but it can help reduce merge conflicts in specific scenarios.